Formula/vpn-slice.rb

```diff
@@ -0,0 +1,24 @@
+class VpnSlice < Formula
+  include Language::Python::Virtualenv
+
+  desc "Vpnc-script replacement for easy and secure split-tunnel VPN setup"
+  homepage "https://github.com/dlenski/vpn-slice"
+  url "https://github.com/dlenski/vpn-slice/archive/v0.13.tar.gz"
+  sha256 "08f62c688b8e5f0a1f643593bd6d29c38ab92e3714a5108bfde762a3cdaf33df"
+  head "https://github.com/dlenski/vpn-slice.git"
+
+  depends_on "python"
```

Comment:
```suggestion
  depends_on "python@3.8"
```

Reply:
@Bo98 anything else that needs to be done?

Reply:
@SMillerDev `vpn-slice` really only needs Python 3.3+

Reply:
That doesn't matter, anything in homebrew should always be using the latest version.

Reply:
@SMillerDev OK, just seems weird to pin it to a specific version when it's not needed at all. I guess I need to change it to get this PR merged?

Reply:
We're actually moving to a system where we version the more problematic formula by default. You're actually ahead of the default python.